extensions/panache/hibernate-reactive-panache/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,12 @@
+---
+name: "Hibernate Reactive with Panache"
+metadata:
+  keywords:
+  - "hibernate-reactive-panache"
+  - "panache"
+  - "hibernate"
+  - "jpa"
+  guide: "https://quarkus.io/guides/hibernate-orm-panache"
```

Comment:
You won't have a specific guide?

Reply:
We will, but there's no guide yet, as is the case for HR.

Reply:
In that case, better remove the guide than pointing to an inappropriate one.

Reply:
OK.